stable/redis/templates/redis-master-svc.yaml

```diff
@@ -24,11 +21,6 @@ spec:
   - name: redis
     port: 6379
     targetPort: redis
-  {{- if .Values.metrics.enabled }}
```

Comment:
Why do you remove this section? It still applies

Reply:
It does, but the exporting here is moved to the 'metrics-svc' as per https://github.com/kubernetes/charts/pull/5495 - no need for it to be in both places.